src/components/Pages/Contact.vue

```diff
@@ -1,23 +1,153 @@
+            <div>
+              <h2 class="mt-10 mb-2">Contact Us</h2>
+              <v-text-field
+                id="Name"
+                v-model="name"
+                :rules="[() => !!name || 'This field is required', v => v.length <= 300 || 'Max 300 characters']"
+                type="text"
+                label="Your Name"
+              />
+              <v-text-field
+                id="contactDetail"
+                v-model="contactDetails"
+                :rules="[v => v.length <= 300 || 'Max 300 characters']"
+                label="How can we reach you? (optional)"
+                hint="For example: email address, Telegram handle, phone number"
+              />
+              <v-select
+                :items=items
+                v-model="subject"
+                label="What's the main reason for your message?"
+              />
+              <v-textarea
+                id="inputMessage"
+                counter
+                v-model="message"
+                :rules="[v => v.length <= 10000 || 'Message must be 10000 characters or less']"
+                auto-grow
+                label="Message"
+              />
+              <div align="right">
+                <v-btn color="primary" @click="send">
+                  SEND MESSAGE
+                </v-btn>
+              </div>
+            </div>
           </v-col>
         </v-row>
+          <v-snackbar color="success" elevation="24" v-model="successMessage">
+            Your message has been sent. Thanks!
+            <template v-slot:action>
+              <v-btn
+                text
+                variant="text"
+                @click="closeAlert"
+              >
+                Close
+              </v-btn>
+            </template>
+          </v-snackbar>
+          <v-snackbar color="error" elevation="24" v-model="errorMessage" multi-line>
+            Something went wrong with sending your message. Please try again.
+            <template v-slot:action>
+              <v-btn
+                text
+                variant="text"
+                @click="closeAlert"
+              >
+                Close
+              </v-btn>
+            </template>
+          </v-snackbar>
       </v-container>
     </v-main>
 </template>
 
 <script>
 export default {
-  name: 'Contact',
-  created () {
-    window.location.href = 'https://wikiba.se/contact/'
+  data: () => ({
+    items: [
+      { text: 'General question', value: 'general-question' },
+      { text: 'Feature request', value: 'feature-request' },
+      { text: 'Report a problem', value: 'report-a-problem' },
+      { text: 'Give feedback', value: 'give-feedback' },
+      { text: 'Other', value: 'other' }
+    ],
+    name: '',
+    contactDetails: '',
+    subject: '',
+    message: '',
+    successMessage: false,
+    errorMessage: false
+  }),
+  methods: {
+    closeAlert () {
```

Comment:
This method will close the snackbar in both case (success and error). I don't know if it'll be better if I have 2 separate methods for each case? @tarrow

Reply:
I think there isn't a massive motivation to go in either direction. Right now I think this is fine :)